### DISCUSSION TITLE
New logo

### DISCUSSION THREAD

**Discussion by arekf** [yattee/yattee#432] [General]
As you might know, we need to change Yattee logo because Apple is having issues accepting builds with it (they say it is too similar to YouTube which is kind of a joke I guess). 

I am looking for a volunteer to create a new logo. There are no other requirements than:
1) it has to be delivered in SVG format,
2) it gives a hint it is video/music app

Does not have to be similar to current one, not even the colors.

I am looking for somebody willing to help me with this as I am not graphic designer and I prefer to spend time fixing more issues before next App Store release.

Thanks!

**Comment by CarolusVitalis** [comment]
I got inspired by a user from Discord and designed this icon. Let me know what you think
![Yattee](https://github.com/yattee/yattee/assets/70665845/71077784-8869-40ec-86e2-2ad54e06aba1)
